src/parser/jobs/gnb/modules/SonicBreak.tsx

```diff
@@ -0,0 +1,175 @@
+		}
+	}
+
+	private breakEnder(event: Event) : void {
+		const lastSonicBreak = this.lastSonicBreakApplication
+
+		if (lastSonicBreak != null && lastSonicBreak.end == null) {
+
+			lastSonicBreak.end = event.timestamp
+
+			if (lastSonicBreak.totalTicks < MAX_TICKS) {
+				lastSonicBreak.isMissingTicks = true
+			}
+		}
+	}
+
+	private onComplete() {
+		const badSonicBreaks = this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks)
+		if (badSonicBreaks.length > 0) {
+			const missedTicks = badSonicBreaks.reduce((acc, val) => acc + (MAX_TICKS - val.totalTicks), 0)
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.SONIC_BREAK.icon,
+				content: <Trans id="gnb.sonic-break.suggestions.missing-ticks.content">
+					One or more of your <DataLink action = "SONIC_BREAK"/> dots had ticks that did not deal damage. </Trans>,
+				tiers: {
+					1: SEVERITY.MINOR,
+					5: SEVERITY.MEDIUM,
+					10: SEVERITY.MAJOR,
+
+				},
+				value: missedTicks,
+				why: <Trans id = "gnb.sonic-break.suggestions.missing-ticks.why"> You wasted {missedTicks} ticks of <DataLink action = "SONIC_BREAK"/> </Trans>,
+			},
+			))
+		}
+	}
+
+	override output() {
+		if (this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks).length === 0) { return }
+
+		const missedTickWindows = this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks)
+
+		const tickTable = <Table collapsing unstackable compact="very">
+			<Table.Header>
+				<Table.Row>
+					<Table.HeaderCell><Trans id="gnb.sonic-break.ticktable">Timestamp</Trans></Table.HeaderCell>
+					<Table.HeaderCell><Trans id="gnb.sonic-break.ticktable.drift-header">Missing Ticks</Trans></Table.HeaderCell>
+					<Table.HeaderCell></Table.HeaderCell>
+				</Table.Row>
+			</Table.Header>
+			<Table.Body>
+				{missedTickWindows.map(window => {
+					return <Table.Row key={window.start}>
+						<Table.Cell>{this.parser.formatEpochTimestamp(window.start)}</Table.Cell>
+						<Table.Cell>
+							<Trans id="sonic-break.tick-issue">
+								<DataLink action = "SONIC_BREAK"/> had {MAX_TICKS - window.totalTicks} ticks deal no damage.
+							</Trans>
+						</Table.Cell>
+						<Table.Cell>
+							<Button onClick={() => {
+								if (window.end) {
+									this.timeline.show(window.start - this.parser.pull.timestamp, window.end - this.parser.pull.timestamp)
+								}
+							}}>
+								<Trans id="gnb.sonic-break.ticktable.timelinelink-button">Jump to Timeline</Trans>
+							</Button>
+						</Table.Cell>
+					</Table.Row>
+				})}
+			</Table.Body>
+		</Table>
+
+		return <Fragment>
+			<Message>
+				<Trans id="gnb.sonic-break.accordion.message">
+					<DataLink action = "SONIC_BREAK"/> is a {this.data.statuses.SONIC_BREAK.duration / 1000} second dot that should tick {MAX_TICKS} times for damage, if you are finding that the target is going invulnerable, consider moving it forward in your <DataLink action = "NO_MERCY"/> windows.
```

Comment:
_i18n:_ `{MAX_TICKS} times` should be made into an embedded plural to accommodate languages that use more than just the `one` and `other` pluralization buckets.

Reply:
My i18n knowledge is failing me on this one. This statement is always going to be plural in English, do you want me to make it a pluralizaton bucket even though it's always plural? I figured other lanaguages would be able to just adjust it the sentence to handle their langauges way of saying multiple. (SE will now make a dot that ticks once just to spite me)

Reply:
The problem is that the translators don't have the context of "this value is always 10". If we just hardcoded it as "10" in the string then it'd be fine without the plural, but making it a variable (even if it's technically static) complicates things.

Reply:
my laziness has come back to create extra work once again...

so do you want:
<Plural value={MAX_TICKS} one = "# time" other="# times"/>
or
<Plural value={MAX_TICKS} many = "# times"/>?

I've done the former, but not sure if the latter is what you are getting at.